recipe/meta.yaml

```diff
@@ -1,15 +1,15 @@
-{% set version = "3.6.2" %}
+{% set version = "3.6.3" %}
```

Comment:
This should be 3.7.0 due to the small breaking change

Reply:
please feel free to cut a new release then

Reply:
It would be nice to give others a voice, agree or disagree, and than act. Merging this and outsourcing the work, which is not much with, only makes people upset. I agree with @isuruf here, there are breaking changes and it would have been better to have it 3.7.0. Cutting a new release with the same functionality as 3.6.3 is silly now that this is merge.

TL;DR before asking "send a PR, do the work, etc" read, comment, give input. It is not too much to ask.